lib/views/auth/auth_page_form.dart

```diff
@@ -0,0 +1,298 @@
+            },
+            onSubmitted: (_) {
+              submit();
+            },
+            type: LenraTextFormFieldType.email,
+            size: LenraComponentSize.large,
+            autofillHints: [AutofillHints.email],
+          ),
+          //------Password------
+          LenraTextFormField(
+            label: 'Your password',
+            obscure: _hidePassword,
+            hintText: 'Password',
+            onChanged: (String value) {
+              password = value;
+            },
+            onSubmitted: (_) {
+              submit();
+            },
+            type: LenraTextFormFieldType.password,
+            size: LenraComponentSize.large,
+            autofillHints: [AutofillHints.password],
+            onSuffixPressed: () {
+              setState(() {
+                _hidePassword = !_hidePassword;
+              });
+            },
+          ),
+          LenraFlex(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              LenraCheckbox(
+                value: keep,
+                onPressed: (value) {
+                  setState(() {
+                    keep = value!;
+                  });
+                },
+              ),
+              LenraText(
+                text: "Keep me logged in",
+                style: themeData.lenraTextThemeData.disabledBodyText,
+              ),
+            ],
+          ),
+          LenraFlex(
+            direction: Axis.vertical,
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              LoadingButton(
+                text: "Log in",
+                onPressed: () {
+                  submit();
+                },
+                loading: isLogging,
+              ),
+              Container(
+                padding: EdgeInsets.only(top: 10.0),
+                child: RichText(
+                  text: TextSpan(
+                    text: "I forgot my password",
+                    style: themeData.lenraTextThemeData.blueBodyText,
+                    recognizer: TapGestureRecognizer()
+                      ..onTap = () {
+                        CommonNavigator.go(context, CommonNavigator.lostPassword);
+                      },
+                  ),
+                ),
+              ),
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  void submit() {
+    if (_formKey.currentState!.validate()) {
+      AuthModel authModel = context.read<AuthModel>();
+      if (isRegisterPage) {
+        authModel.register(email, password).then((_) {
+          GoRouter.of(context).go(context.read<AuthModel>().redirectToRoute ?? '/');
+        }).catchError((error) {
+          logger.warning(error);
+        });
+      } else {
+        authModel.login(email, password, keep).then((_) {
+          GoRouter.of(context).go(context.read<AuthModel>().redirectToRoute ?? '/');
+        }).catchError((error) {
+          logger.warning(error);
+        });
+      }
```

Comment:
You could avoid duplication:
```suggestion
      final futur = isRegisterPage ? authModel.register(email, password) : authModel.login(email, password, keep);
      futur
        .then((_) {
          GoRouter.of(context).go(context.read<AuthModel>().redirectToRoute ?? '/');
        }).catchError((error) {
          logger.warning(error);
        });
```